etl/src/main/scala/etl/RecipeParsing.scala

```diff
@@ -61,14 +70,20 @@ object RecipeParsing {
       if (withoutServingCount(0).nodeName != withoutServingCount(1).nodeName) {
         val title = text(withoutServingCount(0))
         val ingredients = withoutServingCount.drop(1).map(text)
-        Some(IngredientList(title = Some(title), ingredients))
+        Some(IngredientsList(title = Some(title), ingredients))
       } else {
         val ingredients = withoutServingCount.map(text)
-        Some(IngredientList(title = None, ingredients))
+        Some(IngredientsList(title = None, ingredients))
       }
     }
   }
 
+  def guessSteps(body: Seq[Element]): Option[Seq[String]] = {
+    val candidates: Seq[Element] = body.tail.filter(NumberedParagraphs.matches)
```

Comment:
This needs to be deleted. Previously I wasn't using numbers to discern method stops. `tail` was to excluded the intro text and if there wasn't intro text it would just exclude an ingredient.